RELEASE-NOTES.md

```diff
@@ -1,5 +1,98 @@
 # Prefect Release Notes
 
+## Release 2.13.8
+
+### Introducing `flow.deploy`
+
+When we released `flow.serve`, we introduced a radically simple way to deploy flows. Serving flows is perfect for many use cases, but the need for persistent infrastructure means serving flows may not work well for flows that require expensive or limited infrastructure.
+
+We're excited to introduce `flow.deploy` as a simple transition from running your served flows on persistent infrastructure to executing your flows on dynamically provisioned infrastructure via work pools and workers. `flow.deploy` ensures your flows execute consistently across environments by packaging your flow into a Docker image and making that image available to your workers when executing your flow.
+
```

Comment:
Seeing this as a user of prefect deploy, maybe mention that this is the Python-based deployment experience companion to prefect deploy?

Reply:
Something like:

```We're excited to introduce `flow.deploy`, the Python-based deployment experience companion to `prefect deploy`. A simple transition from running your served flows on persistent infrastructure to executing your flows on dynamically provisioned infrastructure via work pools and workers. `flow.deploy` ensures your flows execute consistently across environments by packaging your flow into a Docker image and making that image available to your workers when executing your flow.
``` ?

Reply:
hm, sorry, I think it's a companion to `flow.serve`. I'm not sure what the right word is

Reply:
Yeah, I was a bit confused. Maybe just literally '... introduce `flow.deploy`, a companion to `flow.serve`, a simple ...`

Reply:
@desertaxle ^^ thougths?

Reply:
I'm not sure how to mention `prefect deploy` here. The message would be something along the lines of 'you can keep using `prefect deploy`', but I'm not sure if explicitly saying that will take the focus off of `flow.deploy`. 

I lean towards not mentioning `prefect deploy` here unless someone else feels strongly that we should.

Reply:
That works for me.